web-gateway/src/main/twirl/views/createItem.scala.html

```diff
@@ -8,7 +8,7 @@
 
     <h2>@message("createItem")</h2>
 
-    @foundationForm(itemForm, routes.ItemController.createItem()) {
+    @foundationForm(itemForm, play.api.mvc.Call("POST", "/item")) {
```

Comment:
I mean, I understand the change but the final code due to maven-play limitations is a step back. :-(